platforms/org.eclipse.mita.platform.arduinouno/ARD_SDK/ARD_LED.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * ARD_LED.h
+ *
+ *  Created on: 24.05.2018
+ *      Author: rherrmann
+ */
+
+#ifndef SRC_GEN_BASE_ARD_LED_H_
+#define SRC_GEN_BASE_ARD_LED_H_
+
+#include "ARD_Retcode.h"
+#include <avr/io.h>
+
+Retcode_T ARD_LED_Connect(void);
+
+Retcode_T ARD_LED_Enable(uint8_t color);
+
+Retcode_T ARD_LED_Switch(uint8_t color, uint8_t on);
+
+enum {
```

Comment:
Please give these types a name and use them in your functions.

Reply:
Color_T und Command_T